src/TuDa.CIMS.Web/Services/IPurchaseApi.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Text.Json;
+using Refit;
+using TuDa.CIMS.Shared.Attributes.ServiceRegistration;
+using TuDa.CIMS.Shared.Entities;
+using TuDa.CIMS.Web.Extensions;
+using TuDa.CIMS.Shared.Dtos;
+using TuDa.CIMS.Shared.Attributes.ServiceRegistration;
```

Comment:
Remove this

Reply:
Why? How do I get the "Purchase" type then for example?

Reply:
I only meant the
```cs
using TuDa.CIMS.Shared.Attributes.ServiceRegistration;
```
because its doubled